src/main/features/certificateOfService/routes/document-download.ts

```diff
@@ -0,0 +1,46 @@
+import * as express from 'express'
+import { Paths } from 'certificateOfService/paths'
+import DocumentsClient from 'app/documents/documentsClient'
+import * as http from 'http'
+import * as HttpStatus from 'http-status-codes'
+import { UploadedDocument } from 'claims/models/uploadedDocument'
+import User from 'idam/user'
+
+function getType (documents: UploadedDocument[], fileName: string): string {
+  return documents.find((document) => document.fileName === fileName).fileType
+}
+
+export default express.Router()
+  .get(Paths.documentDownloadPage.uri, (req: express.Request, res: express.Response, next: express.NextFunction) => {
+    DocumentsClient.getBinaryUrl(res.locals.user.bearerToken, req.query.selfPath)
```

Comment:
I thought the plan was to pass the array index to this path and load the document details from the draft store?

Reply:
I'm going to do that when I do the limiting file type story. this needs to be merged so vivek can start his integration testing